test/logging/logging.go

```diff
@@ -24,66 +24,29 @@ import (
 	"flag"
 	"os"
 	"strconv"
-	"strings"
 	"sync"
-	"time"
 
-	"github.com/davecgh/go-spew/spew"
-	"go.opencensus.io/stats/view"
-	"go.opencensus.io/trace"
+	"go.opentelemetry.io/otel/trace"
+	"go.opentelemetry.io/otel/trace/noop"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 	"k8s.io/klog/v2"
 )
 
-const (
-	// 1 second was chosen arbitrarily
-	metricViewReportingPeriod = 1 * time.Second
-
-	// prefix attached to metric name that indicates to the
-	// ExportSpan method that span needs to be emitted.
-	emitableSpanNamePrefix = "emitspan-"
-)
+var noopTracer = noop.NewTracerProvider().Tracer("knative.dev/pkg/test/logging")
 
 // FormatLogger is a printf style function for logging in tests.
 type FormatLogger func(template string, args ...interface{})
 
-var exporter *zapMetricExporter
-
-// zapMetricExporter is a stats and trace exporter that logs the
-// exported data to the provided (probably test specific) zap logger.
-// It conforms to the view.Exporter and trace.Exporter interfaces.
-type zapMetricExporter struct {
-	logger *zap.SugaredLogger
-}
-
-// ExportView will emit the view data vd (i.e. the stats that have been
-// recorded) to the zap logger.
-func (e *zapMetricExporter) ExportView(vd *view.Data) {
-	// We are not currently consuming these metrics, so for now we'll juse
-	// dump the view.Data object as is.
-	e.logger.Debug(spew.Sprint(vd))
-}
-
 // GetEmitableSpan starts and returns a trace.Span with a name that
 // is used by the ExportSpan method to emit the span.
 //
 //nolint:spancheck
-func GetEmitableSpan(ctx context.Context, metricName string) *trace.Span {
-	_, span := trace.StartSpan(ctx, emitableSpanNamePrefix+metricName)
+func GetEmitableSpan(ctx context.Context, metricName string) trace.Span {
+	_, span := noopTracer.Start(ctx, metricName)
```

Comment:
Why is this always `noopTracer`?  Is that because this is in the testing package?

Maybe put a comment on line 36 about the why of `noopTracer`.